pkgs/top-level/all-packages.nix

```diff
@@ -6026,6 +6026,9 @@ in
   nomad_0_12 = callPackage ../applications/networking/cluster/nomad/0.12.nix {
     buildGoPackage = buildGo114Package;
   };
+  nomad_1_0 = callPackage ../applications/networking/cluster/nomad/1.0.nix {
+    buildGoPackage = buildGo114Package;
```

Comment:
For 1.0 this should be bumped to 1.15 instead, as that's what upstream is using for 1.0. I'm not sure what the preference is on letting it use default `buildGoPackage` vs. setting it to `buildGo115Package` - personally I think the second makes more sense since this shouldn't be later automatically bumped to a newer version of go.

Reply:
Done. Thanks for the feedback.